admin-ui/plugins/services/Components/Configuration/CacheInMemory.test.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import CacheInMemory from "./CacheInMemory";
+import { createStore, combineReducers } from "redux";
+import { Provider } from "react-redux";
+import i18n from "../../../../app/i18n";
```

Comment:
No need for a semi-colon after each instruction, refer to other pages

Reply:
Fixed